assets/css/app.scss

```diff
@@ -2,15 +2,17 @@
 @import "../node_modules/nprogress/nprogress.css";
 @import "_bootstrap_custom.scss";
 @import "../node_modules/bootstrap-icons/font/bootstrap-icons.css";
+@import "_index_component.scss";
```

Comment:
the naming is not really clear, at least prefix it with idea_

Reply:
Done. Moved it to assets/css/live/idea_live/...